src/hardware/targets.json

```diff
@@ -1234,6 +1242,16 @@
                 "prior_target_name": "Jumper_AION_NANO_2400_TX"
             }
         },
+        "tx_900": {
+            "t-20": {
+                "product_name": "Jumper AION T-20 900M TX",
+                "lua_name": "T-20 900 TX",
+                "layout_file": "Jumper T-20 900.json",
+                "upload_methods": ["uart", "wifi", "etx"],
+                "platform": "esp32",
+                "firmware": "Unified_ESP32_2400_TX"
```

Comment:
Did you flash this?

Reply:
That's a mistake! 900 json itself is tested, but not targets.json.. I'll correct soon

Reply:
Typo corrected. Tested with actual flashing process (wifi & etx passthrough)